modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/utilities/forms/DDMFormHandler.js

```diff
@@ -28,7 +28,7 @@ class DDMFormHandler {
 	}
 
 	_attachFormListener() {
-		this.DDMFormInstance.on('fieldEdited', (field) => {
+		this.DDMFormInstance.addEventListener('fieldEdited', (field) => {
```

Comment:
The whole of this strategy will have to be revised as soon as PTR-2238 is resolved by the Forms team. It's gonna be a known bug, and since this specific change is non-effective, I'd leave it up to you to either revert it or not.